client/src/pages/OfferHelp.js

```diff
@@ -50,20 +45,20 @@ const Step1 = (props) => {
     updateState({ ...state, answers: updatedAnswers });
     props.update("helpTypeOffered", checkedAnswers);
   };
-  const toggleNone = () => {
-    const newNone = !none;
-    updateState({ ...state, none: newNone });
-    props.update("helpTypeOffered", newNone ? [] : getCheckedAnswers(answers));
-  };
+  // const toggleNone = () => {
```

Comment:
I commented this out instead of removing it all together

Reply:
hmm not sure what is going on here, I remember implementing this in one of the 2 flows for the step wizards, might have been copied here and not needed or i might already have broken the functionality in some places...